serverlessworkflow/modules/ROOT/pages/core/understanding-workflow-error-handling.adoc

```diff
@@ -1 +1,77 @@
-//= Understanding workflow error handling
\ No newline at end of file
+= Understanding workflow error handling
+
+This guide explains how to handle errors that might occur while executing workflow states. Although a non desirable situation, Workflow states, as part of their interaction with external systems, might fail. For these situations, Serverless Workflow specification provides an link:{spec_doc_url}#workflow-error-handling[error handling] mechanism which allows users to deal with these rainy day circumstances.
```

Comment:
I didn't understand the first part correctly.
Handle errors is how to handle errors or handle errors is a method from serverless workflow? (a question that could be asked)